src/components/NavBar.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { NavLink } from "react-router-dom";
+import { SignOutButton } from "../api";
+
+import "./NavBar.css";
+
+export function NavBar() {
```

Comment:
_love_ that this is a new component. Maybe it could called `AuthenticatedNavBar` just to indicate where it should be used? That would open the door for us to eventually have an `UnauthenticatedNavBar`, which might be nice at the end of this project where we're may be interested in adding a "home page" that describes what this app is doing